recipe.yml

```diff
@@ -24,39 +24,44 @@ description: A starting point for further customization of uBlue images. Make yo
 scripts:
 #  - example.sh
 
-# A list of urls of ".repo" files that should be added to your system.
-# This is the proper way to add custom COPR repos to your image.
-# Tip: Use `%FEDORA_VERSION%` instead of static Fedora version numbers, so that
-# your repos automatically use the Fedora version defined in `fedora-version`,
-# which greatly simplifies future maintenance of your custom recipe.
-extrarepos:
-  # Example (which also demonstrates version number expansion):
-  - https://copr.fedorainfracloud.org/coprs/atim/starship/repo/fedora-%FEDORA_VERSION%/atim-starship-fedora-%FEDORA_VERSION%.repo
+# Custom RPM configuration.
+# These changes will be integrated into your custom image at the "system level".
+rpm:
+  # A list of urls of ".repo" files that should be added to your system.
+  # This is the proper way to add custom COPR repos to your image.
+  # Tip: Use `%FEDORA_VERSION%` instead of static Fedora version numbers, so that
+  # your repos automatically use the Fedora version defined in `fedora-version`,
+  # which greatly simplifies future maintenance of your custom recipe.
+  repos:
+    # Example (which also demonstrates version number expansion):
+    - https://copr.fedorainfracloud.org/coprs/atim/starship/repo/fedora-%FEDORA_VERSION%/atim-starship-fedora-%FEDORA_VERSION%.repo
 
-# These RPMs will be removed from the system image. This step happens during
-# image build, before installing custom RPMs (from the "rpm-install" category).
-rpm-remove:
-  # Remove the native firefox (from Fedora) in favor of the Flatpak.
-  - firefox
-  - firefox-langpacks
+  # These RPMs will be installed from your enabled repositories (Fedora's own
+  # repos, extra "repos" you've added, etc). The installation is automatically
+  # performed via rpm-ostree during the image build, and will be pre-installed
+  # at the "system level" in the final image.
+  install:
+    # Needed for yafti (the first boot installer). Remove if you're not using yafti.
+    - python3-pip
+    # GNOME's GTK4 theme, Libadwaita. Already included in Silverblue, but not other spins.
```

Comment:
Mention that yafti won't run without libadwaita installed (was part of the `Needed for yafti` block previously, but now with an explanation for what libadwaita is, the block-effect disappears

Reply:
@EinoHR Oh yeah, nice idea! Will do.